site/search/SearchChartHitThumbnail.scss

```diff
@@ -1,10 +1,12 @@
 .search-chart-hit-thumbnail {
     aspect-ratio: calc($grapher-thumbnail-width / $grapher-thumbnail-height);
+
     width: 100%;
     background-color: $gray-10;
+    container-type: inline-size; // Enable container queries
```

Comment:
Container queries are [not well-supported](https://caniuse.com/css-container-queries-style). Does it fall back gracefully in unsupported browsers?

I see the comment below about `font-size`. It would be nice to document it here as well, whether the fallback works.

Reply:
I just removed this now and added a bit of css in the RichData component instead